go/fn/tests/result_test.go

```diff
@@ -1,117 +1,118 @@
 // Copyright 2021 The Kubernetes Authors.
 // SPDX-License-Identifier: Apache-2.0
 
-package fn
+package tests
```

Comment:
What is the purpose of a `tests` package?

Reply:
it holds unittests that are not planned to be covered by ./examples

Reply:
Removed.